### PR TITLE
Fix #14 add proper Readme for the project

### LINKED ISSUE #14: Add proper Readme for the project
**Describe the supposed change**
Current readme is not informative enough for outsider to understand anything about this project. Readme should be updated to include at least most important aspect and proper description of this project.

**File and section to be updated**
[README.md](https://github.com/softgitron/potku-console/blob/main/README.md)

**Problem with current form**
There isn't enough information in the Readme to gather any valuable insights.

### PR DESCRIPTION
**Pull request by softgitron**
Closes #14. First draft for a better README. Comments welcome!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,69 @@
...
+
+## Status of the project
+
+This is currently very much a WIP-project. Core team is currently working on basic architecture and documentation of the project.
+
+
+
+## What is potku-console
+
+Potku-console is a service control platform that is user for starting services automatically on demand. Potku-console consists of different kind of modules like: core, frontend, service modules and cloud modules that together form the potku-console. This repository concentrates especially on core and frontend parts, because they form the spine of the potku-console, but also most important first party service and cloud modules are developed in this repository.
```
Comment (EddieTheCubeHead) [inline README.md]: ...that is useD for...
...consists of different kindS of modules...
Maybe remove the like before module listing, IMO would make the sentence flow better.
...especially on THE core and frontend parts...
Maybe remove the also from last sentence and add "as well" to the end.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,69 @@
...
+
+
+
+### Pitch for the potku-console
+
+You would like to own a (insert game name here) server where you and your group of Discord friend could play together. You would like to have server with multiple cores and decent amount of RAM from a cloud provider, because the server software is resource hungry and your 10 year old WWW-server computer isn't quite powerful enough. You seek for a good hosting solution, but can't find anything that would fulfill your needs due to tight budget. You ponder that you could save a ton of money in the cloud expenses, if you turn off the server machine when there are no active players on the server. At first idea felt good, but after a minute you figure out that manually controlling the server's power would be impractical, since some of your friends live in different time regions and they like to even after you gone to sleep. If there just would be an easy way to automate this power cycling process...
```
Comment (EddieTheCubeHead) [inline README.md]: ...own aN (insert game here)...
...group of Discord friendS could...
...like to have A server with multiple cores and A decent amount...
Maybe replace "Seek" in "seek for a" with "search" or "look".
...due to A tight budget...
...At first THE idea felt good...
Maybe replace "time regions" with "time zones".
...and they like to PLAY ON THE SERVER even after you HAVE gone to sleep...
Making use of "if only" in the last sentence would be more natural IMO, ie: "if only there was an easy way..."

File: README.md [hunk 3]
```diff
@@ -1,4 +1,69 @@
...
+
+Yes there is a solution, it is called potku-console. Potku-console monitors automatically player counts on the server and shuts it down after last player has left the game. After the server has been shutdown potku-console sets guard for the servers address that waits for the new players and automatically starts up the server when first player arrives. This arrangement saves ton of money and natural resources.
```
Comment (EddieTheCubeHead) [inline README.md]: ...Potku-console AUTOMATICALLY MONITORS player counts...
...after THE last player...
...potku-console sets A guard...
...for the server's address... (missing apostrophe)
...waits for new players... (no THE with plural form)
...when THE first player...
...saves A ton of money... OR ...saves tonS of money...

File: README.md [hunk 4]
```diff
@@ -1,4 +1,69 @@
...
+
+## Installation instructions
+
+*TODO installation instructions*
+
+
+
+## Technologies and architecture
+
+Potku-console uses following technologies:
```
Comment (EddieTheCubeHead) [inline README.md]: ...uses THE following...

File: README.md [hunk 5]
```diff
@@ -1,4 +1,69 @@
...
+* Rust as a main programming language for core and modules
+* Frontend language and technologies yet to be decided
+* [OpenAPI](https://swagger.io/specification/) for API design and REST principles
+* [SQLite](https://www.sqlite.org/index.html) as a database
+* [Docker](https://www.docker.com/) for deployment
+
+More information about [potku-console architecture](https://github.com/softgitron/potku-console/wiki/Components-overview) can be found from project's wiki.
```
Comment (EddieTheCubeHead) [inline README.md]: ...found from THE project's wiki...

File: README.md [hunk 6]
```diff
@@ -1,4 +1,69 @@
...
+
+
+
+## Getting involved and contributions
+
+Awesome to see that you are interested to make this project even better! We are open for all improvements and contributions. See [Contributing wiki-page](https://github.com/softgitron/potku-console/wiki/Contributing) for basic instruction.
```
Comment (EddieTheCubeHead) [inline README.md]: ...interested IN MAKING this...
...for basic instructionS.

**Review by EddieTheCubeHead — CHANGES_REQUESTED** [review]
Left comments with typos, mistakes and suggestions. Suggestios are optional, but won't approve before the typos and mistakes are fixed.

File: README.md (hunk 3 above)
Comment (EddieTheCubeHead) [reply README.md]: no THE with plural form is not grammatically correct advice btw, my bad. The "the" doesn't belong there, but plurality has nothing to do with it.

**Comment by softgitron** [comment]
Thanks for a great review Eddie! I accepted all of your proposals and pushed a new [commit](https://github.com/softgitron/potku-console/pull/15/commits/bc5ab17d6c0a9d21c3908abc45e10f372899e3a5). I have always been especially bad with articles, so it came to me as no surprise that there are a ton of article errors. It is still excellent to get those errors fixed especially on Readme, so that our users don't get irritated at once, when they open our repository's front page.

**Review by EddieTheCubeHead — APPROVED** [review]

**Pull request merged**